rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Include `book & person `  in attr_accessor.
- `attr_accessor :date, :book, :person`